File: zhiyoufy-server-java/zhiyoufy-server/src/main/java/com/example/zhiyoufy/server/domain/dto/jms/JmsJobChildRunResultFull.java
package com.example.zhiyoufy.server.domain.dto.jms;

import java.util.Date;
import java.util.Map;

import lombok.Getter;
import lombok.Setter;

import org.springframework.data.annotation.Id;
import org.springframework.data.elasticsearch.annotations.Document;
import org.springframework.data.elasticsearch.annotations.Field;
import org.springframework.data.elasticsearch.annotations.FieldType;

@Getter
@Setter
@Document(indexName="job-child-run-result")
public class JmsJobChildRunResultFull {
	@Id
	private String id;

	@Field(type= FieldType.Date)
	Date timestamp;

	private long environmentId;

	@Field(type= FieldType.Keyword)
	private String environmentName;

	private long projectId;

	@Field(type= FieldType.Keyword)
	private String projectName;

	private long templateId;

	@Field(type= FieldType.Keyword)
	private String templateName;

	@Field(type= FieldType.Keyword)
	String messageId;

	@Field(type= FieldType.Keyword)
	String runGuid;

	Integer index;
	String endReason;
	String endDetail;

	Boolean endOk;
	Boolean resultOk;
	Boolean passed;

	@Field(type= FieldType.Keyword)
	String jobOutputUrl;

	Map<String, Integer> childStatusCntMap;
}

File: zhiyoufy-web/src/model/dto/jms/JmsJobChildRunResultFull.ts
export interface JmsJobChildRunResultFull {
  id: string;
  timestamp: string;
  environmentId: number;
  environmentName: string;
  projectId: number;
  projectName: string;
  templateId: number;
  templateName: string;
  messageId: string;
  runGuid: string;
  index: number;
  endReason: string;
  endDetail: string;
  endOk: boolean;
  resultOk: boolean;
  passed: boolean;
  jobOutputUrl: string;
  childStatusCntMap: any;
}
